SPECIFICATION.md

```diff
@@ -0,0 +1,368 @@
+
+**Additional Required Fields:**
+
+| Field | Type | Description |
+|-------|------|-------------|
+| `versions` | array[string] | Array of simple version strings (e.g., ["1.0.0", "1.1.0"]). Required for version-specific events. |
+| `ranges` | array[string] | Array of VERS-formatted version ranges (e.g., ["vers:npm/>=1.0.0|<2.0.0"]). Required for range-based events. |
+
+Example:
+```json
+{
+  "type": "endOfSupport",
+  "effective": "2024-01-01T00:00:00Z",
+  "versions": ["1.0.0", "1.1.0"],
+  "ranges": ["vers:npm/>=2.0.0|<3.0.0"],
+  "published": "2023-06-01T00:00:00Z",
+  "modified": "2023-06-01T00:00:00Z",
+}
+```
+
+#### endOfGuaranteedSupport
+*Category: Version Event*
+
+Indicates when the manufacturer no longer provides assured services such as technical assistance, user training, repairs, spare parts, and software updates for a component or service. Beyond this period, any support offered is discretionary and may incur extra costs or have restrictions.
+
+**Additional Required Fields:**
+
+| Field | Type | Description |
+|-------|------|-------------|
+| `versions` | array[string] | Array of simple version strings (e.g., ["1.0.0", "1.1.0"]). Required for version-specific events. |
+| `ranges` | array[string] | Array of VERS-formatted version ranges (e.g., ["vers:npm/>=1.0.0|<2.0.0"]). Required for range-based events. |
+
+Example:
+```json
+{
+  "type": "endOfGuaranteedSupport",
+  "effective": "2024-01-01T00:00:00Z",
+  "ranges": ["vers:npm/>=1.0.0|<2.0.0"],
+  "versions": [],
+  "published": "2023-06-01T00:00:00Z",
+  "modified": "2023-06-01T00:00:00Z",
+}
+```
+
+#### endOfLife
+*Category: Version Event*
+
+Indicates when the manufacturer stops distribution of a product after its defined useful life and formally notifies users. It marks the conclusion of the product's lifecycle, following a structured EOL process.
+
+**Additional Required Fields:**
+
+| Field | Type | Description |
+|-------|------|-------------|
+| `versions` | array[string] | Array of simple version strings (e.g., ["1.0.0", "1.1.0"]). Required for version-specific events. |
+| `ranges` | array[string] | Array of VERS-formatted version ranges (e.g., ["vers:npm/>=1.0.0|<2.0.0"]). Required for range-based events. |
+
+Example:
+```json
+{
+  "type": "endOfLife",
+  "effective": "2024-01-01T00:00:00Z",
+  "ranges": ["vers:npm/>=1.0.0|<2.0.0"],
+  "versions": [],
+  "published": "2023-06-01T00:00:00Z",
+  "modified": "2023-06-01T00:00:00Z",
+}
+```
+
+#### endOfProduction
+*Category: Version Event*
+
+Indicates when the manufacturer stops producing a component, often due to newer versions, unavailable parts, market changes, or strategic shifts. Existing units may still be available in warehouses, distribution channels, and for use, but no new units will be manufactured.
```

Comment:
this seems like it only covers physical goods, but i thought we decided v1 would just be for software?

Reply:
removed